drivers/sensor/bmi160/Kconfig

```diff
@@ -208,4 +216,88 @@ config BMI160_GYRO_ODR_3200
 	bool "3200 Hz"
 endchoice
 
+config BMX160_MAG
+	depends on BMX160
+	bool "Magnetometer support"
+
+choice
+	prompt "Magnetometer power mode"
+	depends on BMX160_MAG
+	default BMI160_MAG_PMU_RUNTIME
+
+config BMI160_MAG_PMU_RUNTIME
+	bool "Set at runtime."
+
+config BMI160_MAG_PMU_SUSPEND
+	bool "suspended/not used"
+
+config BMI160_MAG_PMU_NORMAL
+	bool "normal"
+
+config BMI160_MAG_PMU_LOW_POWER
+	bool "low power"
+endchoice
+
+choice
+	prompt "Magnetometer sampling frequency"
+	depends on BMX160_MAG
+	default BMI160_MAG_ODR_RUNTIME
+
+config BMI160_MAG_ODR_RUNTIME
+	bool "Set at runtime."
+
+config BMI160_MAG_ODR_25_32
```

Comment:
Magnetometer is a BMX160 feature, so these Kconfig names are misleading.

Reply:
Ok I try as much as possible to name it properly for folders, files, configs and magnetometer related things.